service/src/main/java/io/camunda/service/MappingServices.java

```diff
@@ -75,5 +75,5 @@ public Optional<MappingEntity> findMapping(final Long mappingKey) {
         .findFirst();
   }
 
-  public record MappingDTO(Long mappingKey, String claimName, String claimValue) {}
+  public record MappingDTO(String claimName, String claimValue) {}
```

Comment:
We keep the key in the DTO for users, primarily for updates as far as I can tell. I think it's actually nicer to keep the key outside so it doesn't have to be nullable, but let's make it consistent across all entities.

Reply:
This isn't from this PR, but I think it'd be good to standardize the name of these objects to either "mapping rule" or just "mapping". We don't have to fix it in this PR though.

Reply:
> We keep the key in the DTO for users, primarily for updates as far as I can tell. I think it's actually nicer to keep the key outside so it doesn't have to be nullable, but let's make it consistent across all entities.

It's not in the scope of this PR, so I'll create a follow-up for this. Same for the second comment